cppcache/src/DistributedSystemImpl.cpp

```diff
@@ -143,11 +148,26 @@ void DistributedSystemImpl::unregisterCliCallback(int appdomainId) {
   }
 }
 
+std::string DistributedSystemImpl::getThreadName() {
```

Comment:
You can use `const std::string&` once you use the `thread_local` implementation.

Reply:
Done.